packages/module/src/Ansible/Ansible.tsx

```diff
@@ -73,24 +92,28 @@ const Ansible: React.FunctionComponent<AnsibleProps> = ({ unsupported, className
   );
 
   return (
-    <i className={ansibleLogoClass} title={unsupported ? "Ansible is not supported" : "Ansible supported" } data-ouia-component-id={ouiaId} {...props}>
-      <svg
-        version="1.1"
-        x="0px"
-        y="0px"
-        viewBox="0 0 2032 2027.2"
-        style={{ enableBackground: 'new 0 0 2032 2027.2' } as React.CSSProperties}
-      >
-        <path
-          className="st0"
-          d="M2030.8,1014.8c0,559.2-453.3,1012.4-1012.4,1012.4C459.2,2027.2,5.9,1574,5.9,1014.8
+    <React.Fragment>
+      {isRHAAP ? RHAAPTechnologyIcon : (
```

Comment:
Here, the title and ouiaID are not passed to the RHAAP version, we should fix that. I also talked with @kaylachumley on Friday and she should give us information about the unsupported state colors for the RHAAP version.